src/Admin/SiteHealth.php

```diff
@@ -80,7 +80,12 @@ public function register() {
 		add_filter( 'debug_information', [ $this, 'add_debug_information' ] );
 		add_filter( 'site_status_test_result', [ $this, 'modify_test_result' ] );
 		add_filter( 'site_status_test_php_modules', [ $this, 'add_extensions' ] );
-		add_action( 'admin_print_styles-site-health.php', [ $this, 'add_styles' ] );
+
+		if ( ! defined( 'HEALTH_CHECK_PLUGIN_VERSION' ) ) {
+			add_action( 'admin_print_styles-site-health.php', [ $this, 'add_styles' ] );
+		} else {
+			add_action( 'admin_print_styles-tools_page_health-check', [ $this, 'add_styles' ] );
+		}
```

Comment:
This is a bit more dry and it won't cause coverage issues:

```suggestion
		$hook_suffix = defined( 'HEALTH_CHECK_PLUGIN_VERSION' ) ? 'tools_page_health-check' : 'site-health.php';
		add_action( "admin_print_styles-{$hook_suffix}", [ $this, 'add_styles' ] );
```